docs/capabilities.md

```diff
@@ -12,7 +12,7 @@ Capability&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbs
 `appium:platformVersion` | The platform version of an emulator or a real device. This capability is used for device autodetection if `udid` is not provided
 `appium:udid` | UDID of the device to be tested. Could be retrieved from Xcode->Window->Devices and Simulators window. Always set this capability if you run parallel tests or use a real device to run your tests.
 `appium:noReset` | Prevents the device to be reset before the session startup if set to `true`. This means that the application under test is not going to be terminated neither its data cleaned. `false` by default
-`appium:fullReset` | Being set to `true` always enforces the application under test to be fully uninstalled before starting a new session. The application data would remain for real devices if the application under test remained on the device under a certain situation. Please check [troubleshooting](troubleshooting.md#clear-the-application-local-data-explicitly-for-real-devices) to clear the application local data for sure. `false` by default
+`appium:fullReset` | Being set to `true` always enforces the application under test to be fully uninstalled before starting a new session. The application data would remain for real devices if the application under test remained on the device under a certain situation. Please check [troubleshooting](troubleshooting.md#clear-the-application-local-data-explicitly-for-real-devices) for more details to clear the application local data for sure. `false` by default
```

Comment:
to clear the application local data for sure -> regarding obsolete application data cleanup on real devices

Reply:
same below